server/app/services/ti/TrustedIntermediaryService.java

```diff
@@ -0,0 +1,80 @@
+package services.ti;
+
+import com.google.common.base.Strings;
+import forms.AddApplicantToTrustedIntermediaryGroupForm;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import javax.inject.Inject;
+import models.TrustedIntermediaryGroup;
+import play.data.Form;
+import repository.UserRepository;
+import services.DateConverter;
+
+public class TrustedIntermediaryService {
+  private final UserRepository userRepository;
+  private final DateConverter dateConverter;
+
+  @Inject
+  public TrustedIntermediaryService(UserRepository userRepository, DateConverter dateConverter) {
+    this.userRepository = userRepository;
+    this.dateConverter = dateConverter;
+  }
+
+  public TIClientCreationResult addNewClient(
+      Form<AddApplicantToTrustedIntermediaryGroupForm> form,
+      TrustedIntermediaryGroup trustedIntermediaryGroup)
+      throws EmailAddressExistsException {
```

Comment:
suggestion (blocking): why not handle `EmailAddressExistsException` here and add an error message to the form?

Reply:
The old code showed a redirection tot he sign-in page on EmailExistsException. I though through about it and decided, if an email account exists, we should let the TI know of the conflicts and may be follow the old code. 
     return redirectToDashboardWithError(
          "Email address already in use.  Cannot create applicant if an account already exists. "
              + " Direct applicant to sign in and go to"
              + " "
              + trustedIntermediaryUrl,
          form);

Reply:
If the email already exists, how does redirecting to the sign-in page help the TI?

Reply:
hmm.. may be... So should I just add this validation in the form and show to the as an form field error?

Reply:
I think that makes the most sense

Reply:
It is done! I have also added a unit test for validating this logic.

Reply:
Yep! Please see comments in the DateConverter.java file

Reply:
Great!